test/models/opensearch_test.rb

```diff
@@ -4,59 +4,66 @@ class OpensearchTest < ActiveSupport::TestCase
   test 'matches citation' do
     os = Opensearch.new
     os.instance_variable_set(:@params, { citation: 'foo' })
-    assert os.matches.select { |m| m[:citation] == 'foo' }
+
+    assert(os.matches.to_json.include?('{"match":{"citation":"foo"}}'))
```

Comment:
I've confirmed that changing something in this test causes it to fail, but not that it was broken before. I'm happy to know that this test syntax is now fail-able, though.